index.html

```diff
@@ -39,5 +39,143 @@ <h1>Hey there. I’m Mariane</br>
+</section>
+<section class="works-title">
+    <h1>My Recent Works</h1>
+    <img src="./images/Indicator.png" alt="Indicator">
+</section>
+<div class="works-section">
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="recent-works">
+    <article class="works-des">
+        <h2>Multi-Post Stories Gain+Glory</h2>
+        <a href="#">Ruby on rails</a>
+        <a href="#">css</a>
+        <a href="#">JavScript</a>
+        <a href="#">html</a>
+        <button type="button"><a href="#">See Project</a></button>
+    </article>
+</section>
+<section class="about-me">
+    <h1>About me</h1>
+</section>
+<p class="about-paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</p>
+    <section class="resume">
+        <button type="button"><a href="#">Get My Resume</a></button>
+    </section>
+    <section class="language">
+        <img src="./images/icon-languages.png" alt="">
+        <h1>Languages</h1>
+        <section class="languages">
+            <a href="#">JavaScript</a>
+            <a href="#">Ruby</a>
+            <a href="#">Html </a>
+            <a href="#">CSS</a>
+        </section>
```

Comment:
The technology categories ("Languages", "Frameworks" and "Skills") as well as the individual list under each category (e.g., "JavasScript", "Ruby", "HTML" and "CSS") are better placed in  a `ul`, each of them in a `li`. Kindly consider using ul and li tags to list all listed items (this and other listed items) for best practices and semantic reasons. :pray:

Reply:
Fixed